### PR TITLE
Upgrade dependency modules for support tf14

### PR DESCRIPTION
**Pull request by guyelia**
## what
* Upgrading dependency modules to versions that supporting Terraform 14

## why
* For using this module with Terraform 14

### CODE REVIEW

**Review by joe-niland — CHANGES_REQUESTED** [review]

**Comment by guyelia** [comment]
@joe-niland thanks! upgraded to v13 format.

**Comment by joe-niland** [comment]
@guyelia thank you - let's see how the tests go

**Comment by joe-niland** [comment]
The tests need updating to be compatible with the automation.

Did you want to have a go with the following?

```
 make init
 make github/init
 cd test/src/
 go mod init github.com/cloudposse/$(cd ../../ && basename `pwd`)
 go mod tidy
 git rm Gopkg.*
 git add go.*
 cp ../../../terraform-null-label/test/src/Makefile 
```

File: context.tf [hunk 1]
```diff
@@ -0,0 +1,169 @@
+#
```
Comment (joe-niland) [inline context.tf:1]: Variables that conflict with this file need to be removed from variables.tf

**Comment by guyelia** [comment]
I gave up :(

**Comment by joe-niland** [comment]
@guyelia some commands are restricted

**Pull request closed without merging**